controllers/orderController.js

```diff
@@ -0,0 +1,36 @@
+const orderService = require('../services/orderService');
+const { catchAsync } = require('../utils/error');
+
+const createOrder = catchAsync(async( req, res )=> {
+  const  userId  = req.user[0].userId;
```

Comment:
`auth middleware`를 통해 반환되는 `req.user`는 현재 userDao에서 반환되는 값으로 되어 있습니다.

토큰 자체를 복호화 했을 때 반환되는, 즉 payload에 담겨있는 유저의 정보를 사용한다면 해당 코드는 다음과 같이 바꿀 수 있습니다.

controller의 동일 부분 수정해주세요~!

 ```suggestion
 // req.user = decoded일 경우
  const  userId  = req.user.userId;
  
  // req.user = decoded.userId일 경우
  const userId = req.user
```